.buildkite/scripts/initCloudEnv.sh

```diff
@@ -43,6 +44,8 @@ terraformDestroy() {
     cd "$(dirname "$tfstate")"
     echo "Uploading terraform.tfstate to Buildkite artifacts"
     buildkite-agent artifact upload "**/terraform.tfstate"
+    buildkite-agent artifact upload "**/.terraform/**"
+    buildkite-agent artifact upload "outputs*.yml"
```

Comment:
Care should be taken to make sure the pipelines are always private. I think that there are sensitive data in the tf state files.